esports_data_from_pandascore/fn-call.json

```diff
@@ -2,10 +2,9 @@
   "code_file": "tmp/x.wasm",
   "function": "scoreFunc",
   "input": {
-    "matches_api_endpoint": "PandaScore API matches endpoint",
```

Comment:
Didn't see the point in making the match API endpoint configurable given that it never changes. Also removing this as a config option made testing easier

Reply:
are we allowed to include the pandascore API endpoint in these examples? I know that was a concern with coingecko, I wasn't sure if this "not specifying the API endpoint" was on purpose or not.

Reply:
It's a publicly documented API so I don't see why we couldn't. Also we do include the CoinGecko API directly in the code as opposed to reading in via a config

Reply:
Kind of a strange, but even though that part of the API is publicly available, it is a violation of Pandascore's ToS for us to re-post parts of their API.  Kind of unfortunate as it creates a friction for their users, but it is nice that we can demo some of our system's features to address that friction. 

To address, please revert the changes to how the pandascore API is managed so that we go back to passing in the as a parameter.

Reply:
Bummer. Removed explicit uses of the actual endpoint and replaced with templated env var instead